### PR TITLE
add kevin

### PR DESCRIPTION
**Pull request by klieret**
@kmnelson

### CODE REVIEW

File: _educators/kevin_nelson.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+# Required:
+title: "Kevin Nelson"     # first (middle) last
+country: US              # your country of residence (2 letters)
+roles: [Instructor]              # subset of [facilitator, instructor, mentor], can stay empty ([])
```
Comment (klieret) [inline _educators/kevin_nelson.md:5]: Currently this is still case sensitive, but I can also fix that later for you after merging this PR :)

Comment (graeme-a-stewart) [reply _educators/kevin_nelson.md:5]: So shall I just merge this now, right?

Comment (klieret) [reply _educators/kevin_nelson.md:5]: Yes, actually it already works as is, will just fix it later for consistency

**Comment by klieret** [comment]
Thanks!

**Pull request merged**